History.md

```diff
@@ -0,0 +1,84 @@
+
+|              |                                                                                                                      |
+| ------------ | -------------------------------------------------------------------------------------------------------------------- |
+| Proposal 2.1 | Give free and easy access to the software. This incentivizes users to test with vendors ICTs and report back issues. |
+
+## Present - OpenSCD the Web Application
+
+OpenSCD in its current form was designed from scratch with the challenges and proposals described in the previous section in mind. OpenSCL being a prototype, it is not surprising OpenSCD was started from scratch to avoid reproducing the mistakes from OpenSCL such as:
+
+- JAXB is not good at dealing with multiple editions of one schema
+- There was no undo and redo capability
+- OpenSCL was not extendable
+- OpenSCL did not have automated tests
+
+It was decided to go with the browser as the browser has a powerful XML engine built in, there were a lot of JavaScript skills available, and it was very easy to ship.
+OpenSCD came with a new plugin infrastructure. This is important, because it opens a new way to overcome challenge 2 like so:
+
+|              |                                                                                                                                              |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| Proposal 2.2 | Allow OpenSCD to be extendable in the easiest way possible to incentivize contributors outside OpenSCD organization to write their own code. |
+
+Although the primary use case in mind was vendors contributing SCL export plugins, it also enabled other use cases as well. It is possible now to extend OpenSCD statically or dynamically without the permission or even the knowledge of the OpenSCD team.
+OpenSCD on the other hand gives a global editing engine with a history, a fast schema validator and SCL knowledge through existing and general plugins.
+
+The next change was triggered by another team working on an SCL editor and collaborating with a vendor. Due to a missing general purpose ???, it was decided to take this opportunity and build a plugin in its own repository, to integrate the needs of the vendor. This particular attempt changed the course of OpenSCD once again. The lesson we learned had nothing to do with SCL per se, but showed us that the thing we want to agree on when we want to agree on software level needs to be more modular than it was. In other words:
```

Comment:
I have added a ??? where I felt the meaning was unclear.

Reply:
I'd go with "tool". Vague enough :D